lib/OpenQA/Plugin/Helpers.pm

```diff
@@ -21,6 +21,7 @@ use warnings;
 use Mojo::ByteStream;
 use db_helpers;
 use Data::Dumper;
+use Carp;
```

Comment:
Funny thing I was removing them when present in whatever file I edited. Ok ok, I'll stop it now. :)